Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -786,7 +786,7 @@ function vtkOpenGLTexture(publicAPI, model) {
     flip = false
   ) => {
     // Now determine the texture parameters using the arguments.
-    publicAPI.getOpenGLDataType(dataType);
+    publicAPI.getOpenGLDataType(dataType, true);
```

Comment:
If this function is called to construct a new texture, feel like the dataType check should always be forced.

Reply:
It's not forced so callers can set the datatype depending on the use case.

Reply:
The `create2DFromRaw` function takes a dataType as an argument but silently rejects it if `openGLDataType` is already set in the texture. I suppose, I can just override that behavior by re-setting the `openGLDataType = 0` before the call to `create2DFromRaw`.

Reply:
Looks like I'm running into a new issue after a rebase with master:
```
[.WebGL-000055D400392300] GL_INVALID_OPERATION: Texture is immutable.
```
So I might have to delete/create the texture in the ImageMapper anyway.